.github/workflows/docs.yml

```diff
@@ -38,7 +43,7 @@ jobs:
         # The message for the commit
         message: "Updating primer.pdf"
         # Name of the branch to use, if different from the one that triggered the workflow
-        ref: docs
+        #ref: docs
```

Comment:
We may need to bump the version to 4.4.0 for this to work.

Reply:
Bumping now to check